test/unit/media/headset.test.ts

```diff
@@ -0,0 +1,159 @@
+import { Observable } from 'rxjs';
+import { SdkHeadset } from "../../../src/media/headset";
+import GenesysCloudWebrtcSdk from '../../../src/client';
+import GenesysCloudWebrtSdk from "../../../src";
+import HeadsetService, { ConsumedHeadsetEvents, VendorImplementation} from 'softphone-vendor-headsets';
+import { SimpleMockSdk } from '../../test-utils';
+
+let sdk: GenesysCloudWebrtSdk;
+let sdkHeadset: SdkHeadset;
+let headsetLibrary: HeadsetService;
+let headsetEvents$: Observable<ConsumedHeadsetEvents>;
+
+describe('SdkHeadset', () => {
+    beforeEach(() => {
+        jest.clearAllMocks();
+        headsetLibrary = HeadsetService.getInstance({ logger: console });
+        headsetEvents$ = headsetLibrary.headsetEvents$;
+        sdk = new SimpleMockSdk() as any;
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/MyPureCloud/genesys-cloud-webrtc-sdk/pullRequest?prid=9082220)
```suggestion
        sdk = new SimpleMockSdk() as unknown;
```